Added Search Functionality only for MovieDB

jrodriguez0202 opened the pull request:
On the search form, type in any movie and it will display the top 10 results with the Title and Overview

jrodriguez0202 committed:
Added Search Functionality only for MovieDB

laurenesco committed:
Added Exception Message Display

laurenesco commented:
Looks great! Awesome job. I just added the ability for an error message to display in case of any exception.

laurenesco committed:
Deleting Debug File

There is a debug file associated with this PR that contains over 2,000 lines of added code. Going to see if we can get by without it to prevent bloat and future merge conflicts.

laurenesco committed:
Delete project.assets.json

laurenesco reviewed (approved):
Deleted a couple of debug/obj files that were bloating the PR. Still tests perfectly.
